src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from "react"
+import styled from "styled-components";
+import { tooltipStyles } from "./Tooltip.styles";
+import { TooltipProps } from "./types";
+
+const TooltipWrapper = styled.div`
+    display: inline-flex;
+    position: relative;
+`
+
+const TooltipChildren = styled.div`
+    ${tooltipStyles.children}
+    background-color: transparent;
+`
+
+const TooltipContainer = styled.div<Pick<TooltipProps, "position">>`
+    ${tooltipStyles.initialStyles}
+    ${tooltipStyles.container}
+    ${(p) => getContainerStyleByPosition(p.position)}
+`
+
+const TooltipBox = styled.span<Pick<TooltipProps, "position">>`
+    ${tooltipStyles.initialStyles}
+    ${tooltipStyles.box}
+
+    ${(p) => getBoxStyleByPosition(p.position)}
+    color: white;
+`
+
+const getBoxStyleByPosition = (position: "top" | "bottom" | "left" | "right" | undefined) => {
+    switch (position) {
+        case "bottom":
+            return `
+            &:after {
```

Comment:
if its one that a line or two can you please wrap these strings in CSS consts and move them to styles.ts 
Thanks and sorry